librz/include/rz_il/definitions/bitvector.h

```diff
@@ -28,7 +28,8 @@ typedef enum {
  *  Ref : https://web.cs.dal.ca/~jamie/UWO/BitVectors/README.html
  */
 typedef struct bitvector_t {
-	ut8 *bits; ///< bits data
+	ut8 *large;
+	ut64 small;
```

Comment:
Please re-add the union and document here the exact condition when large or small is used.